app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Invalid command format."
```

Comment:
There is a missing closing parenthesis for the `ValueError` exception. Ensure that the parentheses are properly closed.